src/bin/readstat.c

```diff
@@ -1,10 +1,31 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <fcntl.h>
-#include <unistd.h>
+#if !defined _MSC_VER
+#   include <unistd.h>
+#endif
 #include <errno.h>
 #include <time.h>
-#include <sys/time.h>
+#if !defined _MSC_VER
+#   include <sys/time.h>
+#else
+#   define READSTAT_VERSION "0.2.0"
```

Comment:
The version should not be hard-coded. Currently we maintain version information in configure.ac.